tests/test_fringing.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from imsim import make_batoid_wcs, CCD_Fringing
+
+def test_fringing():
+    """
+    Test the fringing model. 
+    """
+    # Set a random center ra/dec
+    cra = 54.9348753510528
+    cdec = -35.8385705255579
+    
+    mjd = 60232.3635999295
+    rottelpos = 350.946271812373
+    band = 'y'
+    det_num = 94
+    
+    xarr, yarr = np.meshgrid(range(4096), range(4004))
+
+    
+    # Testing a CCD with an arbitrary location on the focal plane.
+    ra = 54.86
+    dec = -35.76
+    wcs = make_batoid_wcs(ra, dec, rottelpos, mjd, band, 'LsstCam')
+    
+    ccd_fringing = CCD_Fringing(img_wcs = wcs,boresight=[cra, cdec],seed = det_num, spatial_vary= True)
```

Comment:
Normal style puts a space after commas, but not around = signs in a parameter listing.

Reply:
cf. https://peps.python.org/pep-0008/